src/server/managers/AppPermissionManager.ts

```diff
@@ -72,6 +71,7 @@ export class AppPermissionManager {
             console.error(err);
             return false;
         }
+        return true;
```

Comment:
The function does not return a positive response when if finishes, so the condition on [line 32](https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/358/files#diff-56ddd674038dcd71daa839a641d327d12e7e01984cc03bb63ac776d1601eb515L32) is always fulfilled and the actual bridged code does not get executed. Returning a positive response fixes that.